src/styles/code-read.css

```
@@ -39,64 +42,445 @@
+  top: 0;
+  line-height: calc(24 / 20);
+}
+
+.frame53-settings-panel--register-complete .frame53-volume-visible-label {
+  font-size: max(12px, calc(16px * var(--pr-scale)));
+  line-height: calc(19 / 16);
+}
+
+.frame53-settings-panel--register-complete .frame53-volume-range {
+  --frame53-vol-thumb-size: max(20px, calc(24px * var(--pr-scale)));
+  height: calc(28px * var(--pr-scale));
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)